gradle.properties

```diff
@@ -1,7 +1,8 @@
-# https://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/build_number_ranges.html
-# the first year based version (2016.2)
+# used for intellij build, see https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#intellij-extension-version 
 ideaVersion=2020.3
-# see https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#configuration-intellij-extension for list of accepted types
+# specific property for runPluginVerifier 
+sinceIdeaVersion=2020.3
```

Comment:
I doubt this override is necessary. What kind of use case do you have in mind? e.g., Running the verifier locally only against the newer versions?

Reply:
@duemir 
If removed, by default, it will use the `intellij.ideaVersion`.
In my case, with M1 chip, I need to override `intellij.ideaVersion` with a recent one (supporting M1), .i.e:
`gradle runPluginVerifier -PideaVersion=2022.1`

If we remove the current `listProductsReleases.sinceVersion` (and related property `sinceIdeaVersion`, it will use by default value `2022.1`, and only some versions will be tested (missing all `2021.x` and `2020.x`

https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#tasks-runpluginverifier-ideversions
https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#tasks-listproductsreleases-sinceversion

Reply:
I see. OK, maybe we should try the following then:

```suggestion
# specific property for runPluginVerifier to override during runtime `-PsinceIdeaVersion=2022.1`
sinceIdeaVersion=
```